distribution/docs/pom.xml

```diff
@@ -27,7 +27,9 @@
         <maven.build.timestamp.format>yyyy-MM-dd</maven.build.timestamp.format>
         <packaging>pom</packaging>
         <module>docs</module>
-    </properties>
+        <dependency-target-file>distribution/docs/src/main/resources/content/_reference/_dependencyList/maven-dependency-list.txt</dependency-target-file>
+        <dependency-ignored-lines>["", "none", "The following files have been resolved:"]</dependency-ignored-lines>
+        <dependency-ignored-packages>["org.codice.ddf", "ddf"]</dependency-ignored-packages>    </properties>
```

Comment:
Put `</properties>` on a new line. Also how does this work with downstream projects, could codice/alliance append to this doc?

Reply:
I assumed each downstream project would have their own file